packages/libs/wallet-adapter-chainweaver-legacy/package.json

```
@@ -0,0 +1,39 @@
+{
+    "name": "@kadena/wallet-adapter-chainweaver-legacy",
```

Should we name it with `v2` as it applies to that one. Otherwise it might be confusing when there's v3 and maybe another v4 where v2 and v3 are legacy.

```suggestion
    "name": "@kadena/wallet-adapter-chainweaver-v2",
```

---

denied 🗡️